src/pages/FormPage.tsx

```diff
@@ -45,6 +49,13 @@ const FormPage = () => {
     }
   }, [countrySelected]);
 
+  useEffect(() => {
+    if(addressValue !== null && addressValue !== "") {
+      setTextareaDisabled(false)
+    }
+    
+  }, [addressValue]);
+
```

Comment:
```suggestion
  useEffect(() => {
      setTextareaDisabled(addressValue !== null && addressValue !== "")
  }, [addressValue]);

```

Need to reset it to disabled if the user first types in an adress and then removes it. The suggestion above should work